README.md

```diff
@@ -7,7 +7,7 @@ For process overview, please see https://wiki.folio.org/display/TC/RFC+Process
 * Fork the official RFC repo at https://github.com/folio-org/rfcs . This is usually only needed once for any number of RFCs to be submitted.
 * Create a file in the text directory at the top of the forked repo. It should follow the template defined here. Ensure that the file is named appropriately, taking note of the sequence numbers of existing RFCs.
```

Comment:
These two bullets are (I think) word-for-word duplicated on the wiki page.

Reply:
This duplication is intentional in my mind. It is here to provide a complete set of directions for the first stage, while linking back to the wiki for further information needed later in the process.

Reply:
quick start guide vs full documentation essentially?